lib/rule-executors.ts

```diff
@@ -111,3 +106,56 @@ export function undoPrehide(): boolean {
   }
   return !!existingElement;
 }
+
+function querySingleReplySelector(selector: string, parent: ParentNode = document): HTMLElement[] {
+  if (selector.startsWith('aria/')) {
```

Comment:
Should we use [getSelectorType](https://github.com/puppeteer/replay/blob/main/docs/api/README.md#getselectortype)?